starlette/responses.py

```diff
@@ -148,7 +158,7 @@ def delete_cookie(
         self.set_cookie(
             key,
             max_age=0,
-            expires=0,
+            expires=datetime.now(timezone.utc),
```

Comment:
Do we need the field expires at all to delete a cookie?

Reply:
I think so. But let me confirm this one

Reply:
So we don't need an `expires` if we are setting `max_age` to `0`.  
`max_age` takes precedent over `expires`
https://www.rfc-editor.org/rfc/rfc6265#section-4.1.2.2